scripts/run_on_aws.sh

```diff
@@ -13,10 +13,22 @@ fi
 
 set -e
 
-docker pull allenai-docker-private-docker.bintray.io/cuda:8
-
-docker build -t allenai-docker-private-docker.bintray.io/$CONTAINER_NAME . --build-arg PARAM_FILE=$PARAM_FILE
+# Get temporary ecr login.
+eval $(aws --region=us-west-2 ecr get-login)
+
+docker pull 896129387501.dkr.ecr.us-west-2.amazonaws.com/infrastructure/aristo/cuda:8
```

Comment:
This long crazy string is repeated a bunch of times; can you put it in a variable up above?